modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,585 @@
+
+The new default can be over-ridden by explicitly setting a `height` value in the {productname} configuration. So setting this to `200px` will recreate the previous default presentation.
+
+Also, setting the element’s CSS `height` to a value larger than 400px will increase the height of the text entry area. Setting the elmement’s CSS `height` to a value less than 400px will *not* reduce the height of the text entry area, however.
+
+NOTE: While {productname} 6.0 honours the `height` setting in {productname} configuration, to prevent text entry areas from disappearing during re-sizing, the `min-height` value (100px) cannot be over-ridden.
+
+=== formatting user interface name changes
+
+The names of the following user-interface and user experience elements were changed.
+
+[cols="1,1,1"]
+|===
+|ui or ux element |old name              | new name
+
+| Option          | `font_formats`       | `font_family_formats`
+
+| Toolbar item    | `fontselect`         | `fontfamily`
+
+| Menu item       | `fontformats`        | `fontfamily`
+
+| Option          | `fontsize_formats`   | `font_size_formats`
+
+| Toolbar item    | `fontsizeselect`     | `fontsize`
+
+| Menu item       | `fontsizes`          | `fontsize`
+
+| Toolbar item    | `formatselect`       | `blocks`
+
+| Menu item       | `blockformats`       | `blocks`
+
+| Toolbar item    | `styleselect`        | `styles`
+
+| Menu item       | `formats`            | `styles`
+
+| Option          | `lineheight_formats` | `line_height_formats`
+|===
+
+NOTE: The behaviour of these elements is unchanged.
+
+=== primary
+
+the `primary` property was a boolean property available as a basic panel component of `dialogs`.
+
+It has been deprecated and replaced by the new `buttonType` property.
+
+
+[[ui-elements-and-components-tables-positioned-using-margins-not-float]]
+=== Tables are now positioned using `margins`, not `float`
+
+{productname} previously used `float` to position tables to the left or right of the viewport (the working area available to the {productname} editor). This caused multiple layout and UX problems.
+
+For example, consider a {productname} document that included a table that took up less than 100% of the viewport and a paragraph of text below that table. If both the table and the paragraph were selected and their *alignment* was changed to _left_ or _right_, the paragraph of text moved up beside the table.
+
+As another example, consider a {productname} document containing a table that is wider than the viewport. If you selected this table’s contents and set the *alignment* to _right_, the portion of the table that was outside the viewport disappeared to the left, and no scrollbar presented allowing you to bring that portion back into view.
+
+As of {productname} 6.0, tables are positioned using `margins` rather than `float`. This prevents these layout and UX problems occurring.
```

Comment:
```suggestion
As of {productname} 6.0, tables are positioned using `margin` rather than `float`. This prevents these layout and UX problems occurring.
```

Reply:
All `margins` are now `margin` (there were  three such typos all up) throughout the Release notes/Migration guide docs set.